### PR TITLE
Pr-10

### PR DESCRIPTION
**Pull request by CheezItMan**

### CODE REVIEW

File: roman.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      10   => "X",  
+      9    => "IX",  
+      5    => "V",  
+      4    => "IV",  
+      1    => "I"  
+    }
+  end
+
+  def to_roman
+    n = self
```
Comment (ranuseh) [inline roman.rb:22]: More descriptive variable names

**Review by ranuseh — COMMENTED** [review]
Overall all good.

File: roman_test.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
+require 'minitest/autorun'
+require_relative 'roman_numerals'
+
+# Common test data version: 1.0.0 070e8d5
```
Comment (carlabosco) [inline roman_test.rb:4]: Should we add edge cases?

**Comment by paulentine** [comment]
I like how you committed your tests before committing your code. Red-green-refactor!

File: roman.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+class Fixnum
+
```
Comment (cyndilopez) [inline roman.rb]: add constructor, and instantiate number?

File: roman_test.rb [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+class RomanNumeralsTest < Minitest::Test
+  def test_1
+    # skip
+    assert_equal 'I', 1.to_roman
+  end
+
+  def test_2
+    skip
```
Comment (Kimberly-Fasbender) [inline roman_test.rb:12]: Do we want to be skipping these tests? We probably want to comment these out.

File: roman.rb (hunk 3 above)
Comment (cyndilopez) [inline roman.rb:2]: need constructor

File: roman.rb (hunk 1 above)
Comment (shirley1chu) [inline roman.rb:22]: would benefit to leave comment that n is an instance of Fixnum, which gets converted to roman via to_roman

File: roman.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      1    => "I"  
+    }
+  end
+
+  def to_roman
+    n = self
+    result = ""
+    roman.each do |num, letter|
+      result << letter * (n / num)
+      n = n % num
```
Comment (paulentine) [inline roman.rb:26]: Can be slightly shortened with n %= num

File: roman.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
+class Fixnum
+
+  def roman
```
Comment (kdow) [inline roman.rb:3]: Is there a reason this is in a method?

Comment (Kimberly-Fasbender) [inline roman.rb:3]: What do we think about making this an instance variable vs. a function?

**Review by Kimberly-Fasbender — CHANGES_REQUESTED** [review]
